docs/experimental/viz_molecular_demo.py

```diff
@@ -0,0 +1,164 @@
+"""
+======================================================================
+Space filling model actor
```

Comment:
Can you make the title more specific to molecules?

Reply:
Also should we cite some papers? There must be some original papers that promoted these representations.

Reply:
Good catch! The description was for an earlier version of the demo. I updated the title and description.

Reply:
This file is in experimental, should we move it to tutorial @Garyfallidis @SunTzunami @nasimanousheh ?

Reply:
I don't think we should move this to tutorial as this was just for the reviewers/mentors to understand how to work with the module. 
I've made proper tutorials (using `fetcher`) for which I'll create PRs once this PR is merged.